reference/fly-launch.html.md

```diff
@@ -92,10 +92,148 @@ The language- or framework- specific launchers incorporated into `fly launch` ma
+
+### An example of a custom launch
+
+Here's me launching my Flask app that I've written and tested using the local dev server:
+
+```cmd
+fly launch 
+```
+```out            
+Creating app in /Users/chris/FlyTests/hello-gunicorn-flask
+Scanning source code
+Detected a Python app
+Using the following build configuration:
+        Builder: paketobuildpacks/builder:base
+? Choose an app name (leave blank to generate one): 
+```
+
+If I carry on with this buildpack-based build, this story does not end with a working Flask app. Dockerfile-based deployments are much simpler and faster anyway, so I decide to use a Dockerfile. (I hit ctrl-C to stop the launch.)
+
+Conveniently, there's also already a [Dockerfile that works with this app](https://github.com/fly-apps/hello-gunicorn-flask/blob/main/Dockerfile).
+
+With a Dockerfile in my working directory, if I run `fly launch` again, the Dockerfile launcher takes over before the generic Python buildpack one has a chance to.
+
+```cmd
+fly launch
+```
+```out
+An existing fly.toml file was found for app testrun
+? Would you like to copy its configuration to the new app? No
+```
+I have a `fly.toml` in my working directory from cloning [the `fly-apps/hello-gunicorn-flask` repo](https://github.com/fly-apps/hello-gunicorn-flask) to my local machine, but I want to use whatever config `fly launch` gives me, so I answer `No` to that question.
+
+```
+Creating app in /Users/chris/FlyTests/hello-gunicorn-flask
+Scanning source code
+Detected a Dockerfile app
+```
+There's the Dockerfile scanner taking over. Now some general app configuration:
+
+```
+? Choose an app name (leave blank to generate one): testrun
+? Select Organization: Chris (personal)
+Some regions require a paid plan (fra, maa).
+See https://fly.io/plans to set up a plan.
+
+? Choose a region for deployment: Toronto, Canada (yyz)
+```
+
+This is the region where flyctl will start Machines for this app if not otherwise specified. The first deployment will put its Machine(s) in that region.
+
+```
+Created app 'testrun' in organization 'personal'
+Admin URL: https://fly.io/apps/testrun
+Hostname: testrun.fly.dev
+? Would you like to set up a Postgresql database now? No
+? Would you like to set up an Upstash Redis database now? No
+```
+
+`fly launch` offers to provision [Fly Postgres](/docs/postgres/) and [Redis by Upstash](/docs/reference/redis/) databases. I don't need them for my very simple web app.
+
+```
+Wrote config file fly.toml
+? Would you like to deploy now? Yes
+==> Building image
+Remote builder fly-builder-crimson-dew-6190 ready
+...
+```
+
+It's downloaded the new app's configuration into `fly.toml` and offered to deploy straight away. I accept. By default, the Docker build happens on one of Fly.io's remote builder Machines.
+
+`fly launch` always rewrites `fly.toml` even if it's just writing an identical config back into the file. 
```

Comment:
I'm not sure this is still true. 

One thing that's changed is that you can edit `fly.toml` while `fly launch` is waiting at its "deploy now?" prompt and it will use your local file with your custom changes in it -- you couldn't do this in the (more and more distant) past, when I wrote that sentence.

Reply:
OK, we could test the example later? For now, I'm going to remove it since it's ambiguous.